nifi-nar-bundles/nifi-jolt-bundle/pom.xml

```diff
@@ -57,6 +45,22 @@
                 <artifactId>json-utils</artifactId>
                 <version>${jolt.version}</version>
             </dependency>
+            <dependency>
+                <groupId>javax.servlet.jsp</groupId>
+                <artifactId>javax.servlet.jsp-api</artifactId>
+                <version>2.3.3</version>
+            </dependency>
+            <dependency>
+                <groupId>javax.ws.rs</groupId>
+                <artifactId>javax.ws.rs-api</artifactId>
+                <version>2.1.1</version>
+            </dependency>
```

Comment:
These dependencies are the old versions that are no longer current following the upgrade to Spring 6 and Jetty 12. These should be replaced with the new Jakarta equivalent versions.